src/nspawn/nspawn.c

```diff
@@ -187,6 +187,7 @@ static uid_t arg_uid_shift = UID_INVALID, arg_uid_range = 0x10000U;
 static bool arg_userns_chown = false;
 static int arg_kill_signal = 0;
 static bool arg_unified_cgroup_hierarchy = false;
+static bool arg_cgroup_namespace = false;
```

Comment:
oh, i see this now, this is actually not really exposed as argument. Note that the various arg_xyz static variables here are generally the settings passed on the command lines. Given that this isn't one, this should not be called "arg_".

BTW, if this is merely about caching, then I think it would be better to do that internally in cg_ns_supported() with a local static variable, so that every call implicitly gets the cached effect. `cg_unified()` does this internally too, and so does `cg_is_unified_wanted()` for example.